react/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -133,46 +136,60 @@ export default class CustomMonthPicker extends Component {
 
   render() {
     const { value, className, valid, id } = this.props;
+    // eslint-disable-next-line react/prop-types
+    const { label, labelProps, secondaryLabel, tertiaryLabel } = this.props;
```

Comment:
Any reason you are destructuring these props separately to the line above? That coupled with the eslint ignore feels like something isnt playing nice

Reply:
This is how we manage having a central propsType check inside FieldMessage for these props - this is done is all other Components which use FieldMessage.